notifiers/providers/sendgrid.py

```diff
@@ -1,6 +1,44 @@
 from ..core import Provider, Response
 from ..utils import requests
 
+"""A couple of helper functions to help build the schema"""
+
+
+def get_email_object(title: str) -> dict:
```

Comment:
while this is a valid approach, I used a different one in other providers, where I just set the repetitive bit as another class attribute and re-used it. I think for consistency's sake we should still do that, if that's ok with you.

Reply:
I did it this way so that I could parameterize it.  Without that, I either have to make all the email objects exact duplicates - which is not ideal because the functions of each email object are different in the message - or update the schema after the fact, which feels wrong.  Similar problem with the from object - if I can't parameterize whether or not it's a dupe, I can't share the same class attribute.  I definitely want to be consistent, but maybe this case is different?

Reply:
Ok, let's leave this as is then